### ISSUE TITLE
History tab cleared when session persisted

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zaproxy#3207]
Start using ZAP without persisting the session.
Persist it.
The Site tree remains (as it should) but the History disappears.
If you close ZAP and open the session is does appear correctly.

**Comment by kingthorin** [comment]
Note you don't even have to close ZAP to get the history repopulate, you can just re-open the session.

**Comment by kingthorin** [comment]
This seems to be a purely UI issue. I made a lame stand alone script as follows, which causes the table to be redrawn (dunno what better terminology to use).

```javascript
extHist = org.parosproxy.paros.control.Control.getSingleton().
    getExtensionLoader().getExtension(
        org.parosproxy.paros.extension.history.ExtensionHistory.NAME) 
if (extHist != null) {
    extHist.setShowJustInScope(true);
    extHist.setShowJustInScope(false);
}
```

I tried getting the table or the model and doing a `fireTableDataChanged()`, I also tried looping and notifying (like the [source tagger](https://github.com/zaproxy/community-scripts/blob/master/standalone/historySourceTagger.js) script in the community repo does), neither of which seemed to help.
